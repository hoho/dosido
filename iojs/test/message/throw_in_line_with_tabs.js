/* eslint-disable indent */
'use strict';
var common = require('../common');
var assert = require('assert');

console.error('before');

(function() {
	// these lines should contain tab!
	throw ({ foo: 'bar' });
})();

console.error('after');
