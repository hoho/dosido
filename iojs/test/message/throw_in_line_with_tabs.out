before
*test*message*throw_in_line_with_tabs.js:8
	throw ({ foo: 'bar' });
	^
[object Object]
